NAMESPACE

```diff
@@ -24,6 +24,7 @@ exportClasses(Features)
 exportClasses(NumericVariableFilter)
 exportMethods("[")
 exportMethods("names<-")
+exportMethods(aggcounts)
```

Comment:
Suggestion - what about calling that function `featureCounts` instead? The first thing that came to my mind when reading `aggcounts` was *aggressive counts* ;)

Reply:
I initially planned for `aggregationCounts()` because it is called as part of the aggregation function. But given that there is already `SingleCellExperiment::counts()`, `SingleCellExperiment::logcounts()`, `SingleCellExperiment::tpm()`, ... I went for something short and all lower case.  

Open for discussion...